README.md

```diff
@@ -481,7 +481,13 @@ The final new flag is `--kt-file-per-type`.  Thrifty's convention is to generate
 
 `--omit-file-comments` suppresses the comment that prefixes each autogenerated Java file.
 
-Types generated by Thrifty are annotated with `@javax.annotation.Generated` by default; this can be disabled with the `--omit-generated-annotations` flag.
+Types generated by Thrifty are annotated with a `@Generated` annotation by default; this can be disabled with the `--omit-generated-annotations` flag.  The actual type of the annotation may be specified with a `--use-generated-annotation=[jdk8|jdk9|native]` argument.  By default, we assume `jdk8`, i.e. `javax.annotation.Generated`, as this is what's supported in the Android toolchain ()and will be for the foreseeable future).
```

Comment:
I think the `()` here is meant to be just `(`.

Reply:
Yep!  And now the typo is enshrined in v1.0.0 forever :)